### PR TITLE
Cloud migration documentation

### PR DESCRIPTION
**Pull request by hemraker**
Add documentation for migration to the new Umbraco Cloud infrastructure

### CODE REVIEW

**Comment by busrasengul** [comment]
Thank you @hemraker 🙌
We'll get on this asap ✌

File: Umbraco-Cloud/Upgrades/Migrating-To-New-Infrastructure/index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+versionFrom: 7.0.0
+---
+# Migrating to the new Umbraco Cloud infrastruture
+In this article, you’ll find all relevant information to be aware of when a site is migrated from the old to the new Umbraco Cloud infrastructure. This article is kept up-to-date should any new information arise.
+
+## Schedule and timing
+Projects are scheduled for migration through Q3, 2021. When a project is scheduled the project owner and technical contacts will receive direct communication via email 2 weeks in advance with the specific date and time.
+
+### Re-scheduling migration
+In case the timing of the migration schedule does not work for you, it is possible to reschedule by reaching out to...
```
Comment (hemraker) [inline Umbraco-Cloud/Upgrades/Migrating-To-New-Infrastructure/index.md:11]: @mbentancour

**Review by jonat123 — APPROVED** [review]
@hemraker looks good to me, i don't habe any comments 🙂

Let me know once you need it merged in :D

**Comment by hemraker** [comment]
@jonat123 all done - please merge 😊

**Comment by jonat123** [comment]
@hemraker perfect, merging it in :D

File: Umbraco-Cloud/Upgrades/Migrating-To-New-Infrastructure/index.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+### New GIT repository URL
+There will be an improved repository URL you can switch to, but __it is not required as part of the migration__. The GIT repository will still be accessible from the same URL as before and any custom domains assigned to the project will continue to work as expected. 
+
+We’re moving away from the rather verbose GUID-based URLs that are used on Cloud currently to something resembling what you already know from i.e. Github.
+
+    https://my-project-alias.scm.s1.umbraco.io/<GUID>.git 
+
+will become 
+
+    https://scm.umbraco.io/euwest01/my-project-alias.git
```
Comment (mbentancour) [inline Umbraco-Cloud/Upgrades/Migrating-To-New-Infrastructure/index.md:82]: Is this supposed to work @sitereactor ? 
As an example, this is the Git url of a migrated site: https://mbe-test.scm.euwest01.umbraco.io/7e576a93-a7ec-4ee6-bdc3-9e354bc978ba.git
So I believe the only thing that changes is the "s1" now is "euwest01", or am I missing something?

Comment (sitereactor) [reply Umbraco-Cloud/Upgrades/Migrating-To-New-Infrastructure/index.md:82]: This new type of scm hostname hasn't been enabled yet.

Comment (hemraker) [reply Umbraco-Cloud/Upgrades/Migrating-To-New-Infrastructure/index.md:82]: OK, can you please update the information to be correct?

**Pull request merged**